package.json

```diff
@@ -27,9 +27,9 @@
     "@iktakahiro/markdown-it-katex": "4.0.1",
     "@moonrepo/cli": "1.19.3",
     "@openinf/util-text": "1.1.2",
-    "@shopify/cli": "3.53.0",
+    "@shopify/cli": "3.53.1",
     "@shopify/prettier-plugin-liquid": "1.4.1",
-    "@shopify/theme": "3.53.0",
+    "@shopify/theme": "3.53.1",
```

Comment:
can we just get rid of this dependency already?

Reply:
The only one we really need is https://shopify.dev/docs/themes/tools/liquid-prettier-plugin, but going to take that decision @ closing of https://github.com/OpenINF/.github/issues/607